src/guides/v2.3/install-gde/prereq/php-settings.md

```diff
@@ -6,7 +6,7 @@ functional_areas:
   - System
   - Setup
 redirect_from: 
-  - /guides/v2.3/install-gde/prereq/php-settings-ubuntu.html
+  - install-gde/prereq/php-settings-ubuntu.html
```

Comment:
Not sure why you are removing the `/guides/v2.3`. -- this redirect seems to be working fine.

Maybe you meant to remove the redirect section from the  file `src/guides/v2.4/install-gde/prereq/php-settings.md`

Reply:
Yes 2.4 needed changing